Deployment.md

```diff
@@ -0,0 +1,45 @@
+|Web Solutions                             |1.                            |2.          |Why?          |     
+| ---------------------------------------- |:----------------------------:|:----------:|:------------:|
+|Business<br>(Fast Setup)                  |[3rd Party <br> BTCPay Hosts](https://github.com/btcpayserver/btcpayserver-doc/blob/master/ThirdPartyHosting.md)  |[LunaNode <br> Web-Wizard](https://github.com/btcpayserver/btcpayserver-doc/blob/master/LunaNodeWebDeployment.md)* |- Low Difficulty<br>- BTCPayServer Support (1)<br>- Lightning Network (2)     |
+|Cost                                      |Free / Pay                    |10 USD / mo |BTC Accepted  |
+
+**LunaNode Web-Wizard is a VPS solution, deployable from an easy-web interface.*
+
+<br>
+
+|VPS Solutions                          |1.                      |2.          |3.          |Why?     | 
+| ------------------------------------- |:----------------------:|:----------:|:----------:|:-------:|
+|Business<br>(Self Setup)               |[LunaNode](https://medium.com/@BtcpayServer/hosting-btcpayserver-on-lunanode-bf9ef5fff75b)           |[Microsoft<br>Azure](https://github.com/btcpayserver/btcpayserver-doc/blob/master/AzureDeployment.md)    |[Google<br>Cloud](https://github.com/btcpayserver/btcpayserver-doc/blob/master/GoogleCloudDeployment.md) |- Moderate Difficulty<br>- Docker Compose Build<br>- Lightning Network    |
+|Cost                                   |10 USD / mo             |60 USD / mo |70 USD /mo  |BTC Accepted (1)|
+
+*BTCPay can also be deployed on any VPS that meets the [minimal requirements](https://github.com/btcpayserver/btcpayserver-doc/blob/master/FAQ.md#what-are-the-minimal-requirements-to-run-btcpay).*
+
+<br>
+
+|Hardware Solutions                    |1.                          |2.          |Why?      |   
+| ------------------------------------ |:--------------------------:|:----------:|:--------:|
+|Business<br>(Hardware Setup)          |Ready Out Of<br>The Box   |  Installation &<br>Consultation |- Prices Vary By Service<br>- BTCPayServer Support<br>- Lightning Network<br>[Nodl.it](https://www.nodl.it/) / [Lightning in a Box](https://lightninginabox.co/) |
+|                                      |                            |            |BTC Accepted|
+
+<br>
+
+|Developer Solutions                     |                     |                   |              |    
+| -------------------------------------- |:-------------------:|:-----------------:|:------------:|
+|Developer<br>(Testing Setup)            |Manual Install       |Manual Build       |Hardware Build|
+|**Not Recommended<br>For New Users**    |[Install From<br>Command Line](http://blog.sipsorcery.com/?p=1052)|[Build Without<br>Docker Image](https://github.com/btcpayserver/btcpayserver-doc/blob/master/ManualDeployment.md) |[arm32v7](https://hub.docker.com/r/ketominer/btcpayserver-arm32v7/)<br>[BTCPayBox](https://github.com/btcpayserver/btcpayserver-doc/blob/master/HardwareDeployment.md) |
+
```

Comment:
My OCD, would love to see organized list of deployment methods so that a user (if not already clicked) can navigate to an appropriate page.
For example:
Web Deployment
Azure Deployment
Docker Deployment
Google Cloud Deployment
Manual Deployment
Hardware Deployment
Third-Party Hosting
I know this may seem like too much, but structure and navigation are important :)

Reply:
Actually I was wanting to add this too :)